pkg/statistics/handle/ddl/subscriber.go

```diff
@@ -37,9 +37,12 @@ type handler struct {
 
 // NewHandlerAndRegister creates a new handler and registers it to the DDL
 // notifier.
-func NewHandlerAndRegister(statsCache types.StatsCache) {
+func NewHandlerAndRegister(
+	statsCache types.StatsCache,
+	registry *notifier.DDLNotifier,
```

Comment:
I'm not sure why we call it a `registry`.

Reply:
This function is ...AndRegister, so I want to express `DDLNotifier` is used to register. I can change it to `ddlNotifier` if you prefer